sql-cli/noxfile.py

```diff
@@ -16,61 +16,35 @@ def dev(session: nox.Session) -> None:
     development environment to ``.nox/dev``.
     """
     session.install("nox")
-    session.install("-e", "../python-sdk/.[all]")
-    session.install("-e", "../python-sdk/.[tests]")
-    session.install("-e", ".[all]")
-    session.install("-e", ".[tests]")
+    session.install("poetry")
+    session.run("poetry", "install")
 
 
 @nox.session(python=["3.7", "3.8", "3.9"])
-@nox.parametrize("airflow", ["2.2.5", "2.3"])
+@nox.parametrize("airflow", ["2.3"])
```

Comment:
I feel that if the Python SDK is tested with both versions of Airflow, that is probably good enough - unless we find errors. I'm happy to rollback this change.

Reply:
Removed!